### ISSUE TITLE
[Bug] Clarinet Integrate not deploying project contracts in v0.18.0

### ISSUE THREAD

**Issue by homiebtc** [hirosystems/clarinet#145]
## Issue description
After upgrading clarinet to v0.18.0, running `clarinet integrate` does not automatically deploy project contracts in block 2. 
After I downgraded clarinet back to v0.17.0, it deployed project contracts automatically on block 2.

I'm on WSL 2 Ubuntu 20.04 LTS.

## Steps to reproduce the issue

1.  Using the billboard [example](https://github.com/hirosystems/clarinet/tree/main/examples/billboard) and clarinet v0.18.0: 
2.  Running `clarinet integrate` does not deploy contracts
3.  Result of trying to deploy manually: 
```bash
$ clarinet publish --devnet
{"error":"transaction rejected","reason":"FeeTooLow","reason_data":{"actual":468,"expected":598},"txid":"4f7e0c66b839813b72d16da43d3fcf73743091cc3d964abfbd3aa55b23763849"}
```

### Expected
`clarinet integrate` automatically deploys contracts

### Actual
`clarinet integrate` doesn't automatically deploy contract, nor can you do it manually.

**Comment by hugocaillard** [comment]
Hello @lgalabru 
I set `deployment_fee_rate = 10` but `clarinet integrate` still not automatically deploys the contract like it does in v0.17.0
I was able to deploy it with `clarinet publish --devnet` though 👌

**Comment by lgalabru** [comment]
Addressed in `v0.21.0`
